tensorflow_serving/g3doc/setup.md

```diff
@@ -72,7 +72,12 @@ the `tensorflow-serving-api` PIP package using:
 pip install tensorflow-serving-api
 ```
 
-Note: The TensorFlow Serving Python API only supports Python 2.7. It does not support Python 3.
+Note: The TensorFlow Serving Python API
+[is only published for Python 2](https://pypi.python.org/pypi/tensorflow-serving-api/1.0.0),
+but will work for Python 3 if you either build it yourself, or download the Python 2 version,
+unzip it, and copy it into your Python 3 path. There was
```

Comment:
I reopened the issue. Let's remove the last line "There was [an issue tracking publishing the API as Python 3](https://github.com/tensorflow/serving/issues/700), but it was closed."

Reply:
Would you like a link to the issue, without the snark, or should I just leave it off?

Reply:
I'm ok with linking to the issue as a feature request if you'd like. e.g. add something like:
"There's a [feature request](https://github.com/tensorflow/serving/issues/700) to publish a Python 3 package as well".